formulas/functions/statistical.js

```diff
@@ -251,8 +251,30 @@ const StatisticalFunctions = {
 
   MIN: () => {},
 
-  MINA: () => {},
-
+  MINA: (...values) => {
+    const arr = values.map(a => a.value)
+    const flatArr = FormulaHelpers.flattenDeep(arr)
+    const updatedArr = flatArr.map(a => {
+      if(a === true)
+        return 1
+      if(isNaN(a - a))
+        return 0
+      if(!isNaN(parseFloat(a)))
+        return parseFloat(a)
+      if(!isNaN(parseInt(a)))
+        return parseInt(a)
+      if(["boolean", "string"].includes(typeof a))
+        return 0
+      if(typeof a === "number")
+        return a
+      return null
+    })
+    const finalArr = updatedArr.filter((value) => value != null)
+    if(finalArr.length === 0)
+      return 0;
```

Comment:
I think we can avoid this filter by returning `0` rather than `null` in the default case for the `map()` above. This will work because _everything_ non-numeric (except `true`--which is handled--and `DATE`--which is complicated) is converted to `0` for MAXA() anyway.

Better to avoid the extra loop done by `updatedArr.filter()` if we don't need it.

We still want to check for an empty array, but we can now do that before the map.